source/stylesheets/card-back.scss

```diff
@@ -0,0 +1,21 @@
+.card--back{
```

Comment:
If this is a modifier of the `card` component, thoughts on including it all with the other `card` component's styling? It seems arbitrarily removed